doc/rbd/iscsi-target-cli.rst

```diff
@@ -0,0 +1,167 @@
+=============================================================
+Configuring the iSCSI Target using the Command Line Interface
+=============================================================
+
+The Ceph iSCSI gateway is the iSCSI target node and also a Ceph client
+node. The Ceph iSCSI gateway can be a standalone node or be colocated on
+a Ceph Object Store Disk (OSD) node. Completing the following steps will
+install, and configure the Ceph iSCSI gateway for basic operation.
+
+**Requirements:**
+
+-  A running Red Hat Ceph Storage 2.3 cluster
+
+-  The following packages must be installed from a Red Hat Ceph Storage
+   repository:
+
```

Comment:
@dillaman : Thanks for the reminder!  I just got the kernel version from Mike, so I'll update the downstream docs soon.